draft-ietf-quic-http.md

```diff
@@ -857,14 +909,17 @@ HTTP_INTERRUPTED_HEADERS (0x0E):
 : A HEADERS frame without the End Header Block flag was followed by a frame
   other than HEADERS.
 
-HTTP_SETTINGS_ON_WRONG_STREAM (0x0F):
-: A SETTINGS frame was received on a request control stream.
+HTTP_WRONG_STREAM (0x0F):
+: A frame was received on a different stream to where it is permitted.
```

Comment:
"different... to" is British; "different... than" is American.  "Different... from" is neutral.  Maybe we should use that, or just sidestep the issue with "on a stream where it is not permitted."